boolean/boolean.py

```diff
@@ -12,6 +12,13 @@
 import itertools
 import collections
 
+# Python 3
```

Comment:
Alternative would not a `from __future__ import unicode_literals` make this not needed?
Also what is the `# NOQA` about?